### PR TITLE
Partial dictionary update taken from issue #1358

### PR DESCRIPTION
**Pull request by vikivivi**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25132,10 +25386,12 @@ watn->want
 wavelenght->wavelength
 wavelenghts->wavelengths
 wavelnes->wavelines
+wayoints->waypoints
 wayword->wayward
 weant->want, wean,
 weaponary->weaponry
 weas->was
+webaserver->web server
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: IMO `webserver` should also be valid.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -25132,10 +25386,12 @@ watn->want
 wavelenght->wavelength
 wavelenghts->wavelengths
 wavelnes->wavelines
+wayoints->waypoints
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Would you mind also adding `wayoint->waypoint` ?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -25107,10 +25359,12 @@ warinigs->warnings
 warinings->warnings
 warks->works
 warlking->walking
+warmup->warm up
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Should also be `warm-up`  
This is going to show a lot of false positives even though grammatically that is correct.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -24969,17 +25215,21 @@ vissible->visible
...
 viusualisation->visualisation
 viwer->viewer
 viwers->viewers
+vizualization->visualization, virtualization,
 vlarge->large
 vlaue->value
 vlaues->values
 vlues->values
 volatage->voltage
+volatge->voltage
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Would you mind adding a separate line for `volatges->voltages`?

Comment (vikivivi) [reply codespell_lib/data/dictionary.txt]: Added volatges->voltages

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt]: Added warmup->warm up, warm-up,

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt]: Added wayoint->waypoint

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt]: Added webaserver->web server, webserver,

**Comment by luzpaz** [comment]
@vikivivi thank you for following-up. Though why squash the commit and force push it? It makes it _so_ much harder to review what changes you made to the file. You could have introduced more entries and we wouldn't pick up on them. Also it's annoying to scroll back and forth between the collapsed 'Resolved' issue boxes in the PR above and the code in another browser tab. Please don't squash commits, especially for a PR this big. Does that make sense? I know that's something that other repos want people to do, but here in codespell when you're reviewing a single file, squashing is a problem.

**Comment by vikivivi** [comment]
@luzpaz I promised that I did not introduce any more entries. Since I don't have your email. 
Here I shall share you my additional entries only.
This is how you can directly review my PR.

Download from https://www.dropbox.com/s/ikrum6hv4nexiba/dictionary-custom.txt?dl=0
It contain all the additional entries only. No deletion.

$ wget https://www.dropbox.com/s/ikrum6hv4nexiba/dictionary-custom.txt
$ cat codespell_lib/data/dictionary.txt  dictionary-custom.txt > temp.txt
$ LC_ALL=C sort -f -b -u -o dictionary-new.txt temp.txt
$ mv -f dictionary-new.txt codespell_lib/data/dictionary.txt 

Then compare my changes via the newly created codespell_lib/data/dictionary.txt

**Comment by luzpaz** [comment]
@vikivivi thanks :) This is neat workflow that I'm intrigued to try. Woul you mind explaining what's happening in the `LC_ALL=C sort -f -b -u -o dictionary-new.txt temp.txt` line?

**Comment by vikivivi** [comment]
$ cat codespell_lib/data/dictionary.txt dictionary-custom.txt > temp.txt
This line will create a new file temp.txt by combining dictionary.txt and dictionary-custom.txt

$ LC_ALL=C sort -f -b -u -o dictionary-new.txt temp.txt
Since all my custom changes is appended at the end of the file, This line will sort temp.txt according to codespell dictionary sorting technique/rules and output a new file dictionary-new.txt with option '-o'. This is exactly how I maintained my custom dictionary on top of the official release before PR. The sorting is well tested for more than a year already where I maintained my own custom dictionary. I am using Ubuntu LTS 18.04.4

$ mv -f dictionary-new.txt codespell_lib/data/dictionary.txt 
Now update the dictionary.txt with the newly created dictionary-new.txt

**Comment by vikivivi** [comment]
@luzpaz While maintaining my personal custom dictionary. When my script pull upstream latest update to the dictionary, my script will also automatically perform duplication check. BTW, this PR did not have any update changes or deletion. All are new entries.

**Comment by bl-ue** [comment]
Wow, this is really cool! I'll fix the conflicts, review it real quick, and then merge 👍🏻

**Review by bl-ue — APPROVED** [review]

**Pull request merged**